docker/ubuntu-python3/Dockerfile-cuda-10.1

```diff
@@ -58,4 +55,7 @@ RUN tlmgr init-usertree \
 && tlmgr install lm stmaryrd 2>&1 || : \
 && updmap -user
 
+# install contributed Python3 packages
+RUN pip3 install --user --upgrade sphinxcontrib-bibtex 'MDAnalysis>=0.16' 'pint>=0.9'
```

Comment:
have you checked for a package for pint?

Reply:
I had only checked pint's website for the installation instructions, which don't advertise `python3-pint`. Silly me.

Reply:
`python3-pint` is currently version 0.8.1-2 (from 2017) in Ubuntu 18.04, we should probably stick with 0.9 from pip which has fixes for numpy-related compatibility bugs.